game/scripts/npc/abilities/visage_summon_familiars_oaa.txt

```diff
@@ -69,8 +69,6 @@
       {
         "var_type"                                        "FIELD_INTEGER"
         "scepter_total_familiars"                         "3"
-        "LinkedSpecialBonus"                              "special_bonus_unique_visage_6"
```

Comment:
Don't remove this. Visage's scepter in OAA also gives another familiar. I know its not like in vanilla. Its on purpose this way.

Reply:
So you are ok with dmg buff and the 4 familiars? Fine with me in that case.

Reply:
Yeah its fine.